mypy/build.py

```diff
@@ -2421,6 +2415,25 @@ def find_module_and_diagnose(manager: BuildManager,
             raise CompileError(["mypy: can't find module '%s'" % id])
 
 
+def find_added_packages(suppressed: List[str],
+                        manager: BuildManager, options: Options) -> bool:
+    """Find if there are any newly added packages that were previously suppressed.
+
+    Exclude everything not in build for follow-imports=skip.
+    """
+    for dep in suppressed:
+        path = manager.find_module_cache.find_module(dep, manager.search_paths,
+                                                     manager.options.python_executable)
+        if not path or dep in manager.source_set.source_modules:
+            continue
+        if (options.follow_imports == 'skip' and
```

Comment:
It's not clear to me what happens when we have follow_imports as 'skip', and we add a pre-existing `__init__.py` file that used to be skipped to the build. Can you add a comment about this somewhere, assuming that it works?

Reply:
We don't need to add any special logic for this. If a module is added to build, importers will be invalidated by normal mechanism. The test you proposed to add will check this.

(I will add a comment anyway, after I finish with `Final`.)